pkg/controller/operators/catalog/operator_test.go

```diff
@@ -236,14 +238,119 @@ func TestEnsureCRDVersions(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		err := ensureCRDVersions(&tt.oldCRD, &tt.newCRD)
+		err := ensureCRDV1Beta1Versions(&tt.oldCRD, &tt.newCRD)
 		if tt.expectedFailure {
 			require.Error(t, err)
 		}
 	}
 }
 
-func TestRemoveDeprecatedStoredVersions(t *testing.T) {
+func TestEnsureV1CRDVersions(t *testing.T) {
```

Comment:
Can you also do this test where the `currentVersions` are created as `v1beta1` CRDs?

Reply:
yep good idea